tests/benchmark/compute/scenario/test_transaction_types.py

```diff
@@ -72,17 +74,17 @@ def get_single_sender_list(pre: Alloc) -> Generator[Address, None, None]:
         yield sender
 
 
-def get_single_receiver_list(pre: Alloc) -> Generator[Address, None, None]:
+def get_single_receiver_list(
+    pre: Alloc, balance: int
+) -> Generator[Address, None, None]:
     """Get a list of single receiver accounts."""
-    receiver = pre.fund_eoa(0)
+    receiver = pre.fund_eoa(balance)
     while True:
         yield receiver
 
 
-@pytest.fixture
```

Comment:
This could still work as a fixture if we parametrize `balance=0` on the rest of the test cases.